src/create-state.js

```diff
@@ -50,7 +50,7 @@ module.exports = function(vnid, data, lm, error, stale, groupLm, componentName)
                 error: error,
                 stale: stale,
                 groupLm: groupLm,
-                lm: lm || createLm(),
+                lm: (arguments.length > 2) ? lm : createLm(),
```

Comment:
Actually I think this was correct the way it was before, because lm should not be left undefined if data is defined.